### ISSUE TITLE
Screen reader skips characters when using cursor right/left

### ISSUE THREAD

**Issue by egamma** [Microsoft/vscode#2360]
Testing #2258
Position cursor on the line:
>    "files.autoSave": "afterDelay",

Press cursor right. The screen reader reads `"ii` (the f is lost and the i is doubled)

**Comment by alexandrudima** [comment]
I have spent more than 1hr looking into this. IMHO this is a bug inside Chromium. Without making any code changes on my side, here are my observations:
- small files work fine (I could not repro in tiny files with just a few lines of code)
- as soon as a file gets larger and you begin to stress it a little bit by pressing left/right quickly or repeatedly, NVDA starts reading characters +/-1. Sometimes it skips one and reads the same one again
- if you let it breath for a moment, perhaps by clicking on something else and going back it becomes correct again

I have also tried using vanilla electron with a textarea and I could not reproduce in there, so the problem occurs for our textarea (where we use `setSelectionRange`) but not for a normal textarea where all events are handled internally by Chromium.

Given my code does exactly the same thing all the time -- I have added logs in the places I touch the textarea in any way to ensure this --, I think there must be some issue inside Chromium.
